reco_utils/dataset/url_utils.py

```diff
@@ -53,3 +53,15 @@ def maybe_download(url, filename, work_directory=".", expected_bytes=None):
 
     return filepath
 
+
```

Comment:
not sure if we should rename this file from url_utils to download_utils... any preference here?

Reply:
maybe better name to include 'upload', etc. network related utils together? or will that be too broad?

Btw, can we move tqdm to other utils? it is general function to show any progress bars, right? we can even use it for showing model training or evaluation status, not just for downloading status.